<launch>
    <node name="image_source_usbcam" pkg="libuvc_camera" type="camera_node">
        <remap from="image_raw" to="image_source/image_raw" />
        <param name="index" value="0"/>
        <param name="width" value="1280" />
        <param name="height" value="720" />
        <!-- choose whichever uncompressed format the camera supports: -->
        <param name="video_mode" value="mjpeg"/>
        <param name="frame_rate" value="15"/>

        <param name="timestamp_method" value="start"/> <!-- start of frame -->
        <param name="auto_exposure" value="3"/> <!-- use aperture_priority auto exposure -->
        <param name="auto_white_balance" value="false"/>
        <param name="auto_focus" value="false" />
        <param name="auto_hue" value="false"/>

        <!-- name="exposure_absolute" value="0.01"/-->
        <!-- param name="iris_absolute" value="1" /-->
        <param name="focus_absolute" value="10"/>
        <param name="brightness" value="128"/>

    </node>
</launch>
